project.clj

```diff
@@ -30,7 +30,6 @@
             [lein-pprint "1.1.1"]
             [lein-ring "0.9.7"]]
   :main datomic-rest-api.get-handler
```

Comment:
@a8wright yep. I've submitted a PR upstream to the mount repo that should fix the underlying issue (as per ref above)